packages/Operators/src/PointCloud/DTK_SplineInterpolationOperator_impl.hpp

```diff
@@ -237,14 +239,22 @@ void SplineInterpolationOperator<Basis,DIM>::setupImpl(
     if ( Teuchos::is_null(d_stratimikos_list) )
     {
 	d_stratimikos_list = Teuchos::parameterList("Stratimikos");
-	Teuchos::updateParametersFromXmlString(
-	    "<ParameterList name=\"Stratimikos\">"
-	    "<Parameter name=\"Linear Solver Type\" type=\"string\" value=\"Belos\"/>"
-	    "<Parameter name=\"Preconditioner Type\" type=\"string\" value=\"None\"/>"
-	    "</ParameterList>"
-	    ,
-	    d_stratimikos_list.ptr()
-	    );
+
+        d_stratimikos_list->set( "Linear Solver Type", "Belos" );
+        d_stratimikos_list->set( "Preconditioner Type", "None" );
+
+        auto& linear_solver_types_list =
+            d_stratimikos_list->sublist("Linear Solver Types");
+        auto& belos_list = linear_solver_types_list.sublist( "Belos" );
+        belos_list.set( "Solver Type", "Pseudo Block GMRES" );
+        auto& solver_types_list = belos_list.sublist( "Solver Types" );
+        auto& gmres_list = solver_types_list.sublist("Pseudo Block GMRES");
+        gmres_list.set("Convergence Tolerance", 1.0e-10 );
+        gmres_list.set("Verbosity",
+                    Belos::Errors + Belos::Warnings +
+                    Belos::TimingDetails + Belos::FinalSummary +
+                    Belos::StatusTestDetails );
+        gmres_list.set("Output Frequency", 1 );
```

Comment:
Aww man. I'm looking at `Teuchos:: ParameterList` documentation and cannot find anything better. So cumbersome...